src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -390,6 +418,43 @@ def build_cmd(self, env, host, **kwargs):
 
         return tst_cmd
 
+    def convert_xml(self, xml_file):
+        """Modify the xml file"""
+
+        with open(xml_file, 'r') as fd:
+           with open('{}.xml'.format(xml_file), 'w') as ofd:
+              for line in fd:
+                  if self.src_dir in line:
+                      L = re.sub(r'<dir>\/*' + self.src_dir + '\/*',
+                                 r'<dir>',
+                                 line)
+                      ofd.write(L)
+                  else:
+                      ofd.write(line)
+              os.unlink(xml_file)
+
+         return 0
```

Comment:
(pylint-syntax-error) unindent does not match any outer indentation level (<unknown>, line 436)